data_validation/cli_tools.py

```diff
@@ -590,8 +590,12 @@ def _configure_row_parser(
     required_arguments.add_argument(
         "--primary-keys",
         "-pk",
-        required=True,
-        help="Comma separated list of primary key columns 'col_a,col_b'",
+        required=is_custom_query,
+        help=(
+            "Comma separated list of primary key columns 'col_a,col_b'" ""
+            if is_custom_query
+            else " (defaults to table primary key if available)"
```

Comment:
defaults to primary key(s) of the table if available

Reply:
Good points, thanks.